src/components/dropdown/Dropdown.vue

```diff
@@ -336,6 +342,18 @@ export default {
                     zIndex: '99'
                 }
             }
+        },
+
+        activeWindowScrollClose() {
+            // Use a local state to ensure function isnt called more then once
+            let triggered = false
+            const func = () => {
+                if (triggered) return
+                if (this.$refs.optionsDropdown) this.toggle()
```

Comment:
If maxHeight is used, we should not close the dropdown if we're scrolling into it